src/main/resources/default/taglib/w/multiLanguageTextField.html.pasta

```diff
@@ -0,0 +1,97 @@
+<i:arg name="span" type="int" default="6"/>
+<i:arg name="smallSpan" type="int" default="12" description="Defines the span for mobile devices"/>
+<i:arg name="name" type="String" default=""/>
+<i:arg name="fieldName" type="String" default="@name"/>
+<i:arg name="value" type="sirius.db.mixing.types.MultiLanguageString"/>
+<i:arg name="labelKey" type="String" default=""/>
+<i:arg name="label" type="String" default="@i18n(labelKey)"/>
+<i:arg name="forceLabel" type="boolean" default="false"/>
+<i:arg name="helpKey" type="String" default=""/>
+<i:arg name="help" type="String" default="@i18n(helpKey)"/>
+<i:arg name="required" type="boolean" default="false"/>
+<i:arg name="adminOnly" type="boolean" default="false"/>
+<i:arg name="autofocus" type="boolean" default="false"/>
+<i:arg name="type" type="String" default="text"/>
+<i:arg name="id" type="String" default="@generateId()"/>
+<i:arg name="placeholder" type="String" default=""/>
+<i:arg name="tabIndex" type="String" default=""/>
+<i:arg name="defaultLanguage" type="String" default="@NLS.getCurrentLang()"/>
+<i:arg name="validLanguages" type="List" default="@value.getValidLanguages()"/>
+<i:arg name="manageLanguages" type="boolean" default="false"/>
+
+<i:pragma name="description" value="Renders a multi language text input field within a Wondergem template"/>
+
+<i:local name="functionName" value="@('openMultiLanguageModal' + fieldName)"/>
+<i:local name="fieldId" value="@('mls-field-' + id)"/>
+<i:local name="modalId" value="@('mls-modal-' + id)"/>
+
+<div id="@fieldId" class="col-xs-@smallSpan col-md-@span form-group @UserContext.get().signalFieldError(fieldName)">
+    <i:if test="isFilled(label) || forceLabel">
+        <label>
+        <span class="@if (required) { input-required } @if (adminOnly) { admin-link }">
+            @label
+        </span>
+        </label>
+    </i:if>
+    <div class="input-group" id="@('insert-point-' + fieldName)" data-toggle="modal" data-target="#___modalId">
+        <input id="@fieldId"
+               name="@fieldId"
+               type="@type"
+               value="@value.fetchTextOrFallback(defaultLanguage)"
```

Comment:
```suggestion
               value="@value.fetchText(defaultLanguage, 'fallback')"
```
fetchTextOrFallback() throws an exception for fields without fallback. I think the above suggestion returns null instead